examples/run_v2_9_0_all_sets_E3SM_machines.py

```diff
@@ -69,7 +69,7 @@ def run_all_sets():
 
```

Comment:
Do we need a new run script for the latest version of e3sm_diags?

Reply:
I don't think so, we can just use the one run script but keep updating by adding new set.